src/pyscipopt/scip.pxi

```diff
@@ -5439,6 +5533,76 @@ cdef class Model:
         """Get an estimation of the final tree size """
         return SCIPgetTreesizeEstimation(self._scip)
 
+@dataclass
+class Statistics:
+    # Total time since model was created
+    total_time: float
+    # Time spent solving the problem
+    solving_time: float
+    # Time spent on presolving
+    presolving_time: float
+    # Time spent on reading
+    reading_time: float
+    # Time spent on copying
+    copying_time: float
+    # Name of problem
+    problem_name: str
+    # Name of presolved problem
+    presolved_problem_name: str
+    # Dictionary with number of variables by type 
+    _variables: dict
```

Comment:
Why is it private? (goes for the rest of ones prefixed by underscore too. If it's returned by SCIP it should be accessible.

Reply:
The comment should also say what are the keys and values

Reply:
Because of the way .stats is organized, I found it easier to have the variables in a dictionary, which I then unpack in the ```_populate_remaining``` method. Should I just let people access the variables dictionary?

Reply:
No it's ok, we can always make it public later. But I don't think it's necessary, probably if more info is added there we can added more computed properties later